src/App.css

```diff
@@ -0,0 +1,51 @@
+@import url('https://fonts.googleapis.com/css2?family=Fira+Sans:wght@200;300;400;500;600;700;800;900&family=Lato:wght@100;300;400;700;900&display=swap');
+
+*,
+body {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #d6eaf8;
+  padding: 2rem 0;
+}
+
+::-webkit-scrollbar {
+  width: 5px;
+  height: 5px;
+}
+
+::-webkit-scrollbar-track {
+  background: transparent;
+}
+
+::-webkit-scrollbar-thumb {
+  background: rgba(255, 255, 255, 0.5);
+  border-radius: 3px;
+  width: 5px;
+  height: 5px;
+}
+
+::-webkit-scrollbar-thumb:hover {
+  background: #fff;
+}
+
+.data-cont {
+  width: 450px;
+  background-color: #3498db;
+  padding-bottom: 25px;
+  border: 2px solid #d35400;
+  border-radius: 8px;
+}
+
+@media (max-width: 600px) {
+  .data-cont {
+    width: calc(100vw - 40px);
+    margin: 0 auto;
+  }
+}
```

Comment:
- [OPTIONAL] This is the way your application looks on both clickable and non-clickable screens for mobile view

 
![image](https://user-images.githubusercontent.com/77106962/225910686-052095e5-e19c-44ec-a8b7-f1df76030e1c.png)

![image](https://user-images.githubusercontent.com/77106962/225910897-01367f7c-f9a7-484b-80e3-1a4d8d628e0e.png)

#### Kindly notice the white spaces on both screenshots. Ideally, the application should occupy the full width and height on any screen the user uses to view the application. I would advise you to work on the UI and ensure the white spaces are not visible as they are not part of the design. Kindly take a cue from the design template provided for the capstone:

![image](https://user-images.githubusercontent.com/77106962/225911824-926ef6c7-dcf8-46e2-8afd-bb232ca29b9e.png)

#### Notice the way the application looks on the mobile phone in the screenshot above. 😎 💻